### ISSUE TITLE
Best practices with glyphicons

### ISSUE THREAD

**Issue by GarthMartin** [wet-boew/wet-boew#6850]
I was hoping to use glyphicons as buttons in a particular scenario (without text) but was wondering if that is acceptable from an accessibility perspective.  Everything I have found so far (limited search) indicates that the element should be tagged as ```aria-hidden=true``` meaning the screen reader won't read it.  How does one get an alt text type tag applied to the button in this case?

My scenario is thus:

I have a <ul> of simple text items that a use can add/remove from.  They also want to specify the order of items by "moving" them up an down in the list, hence the up and down arrow glyphicons.

Is there a way to have some text be read/displayed if the glyphicon is not available?

Thanks in advance

Garth

**Comment by GarthMartin** [comment]
Sorry it was supposed to read:

" I have a list of simple text items"....

**Comment by LaurentGoderre** [comment]
This is what the `wb-inv` class is for. Here is an example from the multimedia player:

```
<button type="button" class="btn btn-default playpause" aria-controls="wb-auto-2-md" title="Play" data-state-on="Play" data-state-off="Pause">
    <span class="glyphicon glyphicon-play"><span class="wb-inv">Play</span></span>
</button>
```

**Comment by hsrudnicki** [comment]
@GarthMartin @LaurentGoderre : Can this issue be closed?

**Comment by hsrudnicki** [comment]
Make "Going Stale" to close this issue or otherwise?
Either way, please update this issue's status, thank you

**Comment by hsrudnicki** [comment]
@GarthMartin and others: This issue, lingering for 6 months without further discussion/status update(s), labeled "Going Stale" and to remain as opened or if otherwise?

**Comment by hsrudnicki** [comment]
This is closed based on the above. If you think otherwise, please update the status of this again, thank you
